### PR TITLE
Docs/update blog post

### PR DESCRIPTION
**Pull request by LarsDenBakker**
Definitely not a finished product, will add the TS/JSX sections later.

### CODE REVIEW

File: docs/blog/introducing-modern-web/index.md [hunk 1]
```diff
@@ -1,62 +1,64 @@
 ---
-title: Introducing modern web - Developer tools for modern web development.
+title: Introducing: Modern Web.
 published: false
-description: Reexperience the joy when working with the standards based web. Starting off with a test runner which uses multiple browsers in parallel.
+description: Developer tools for modern web development
```
Comment (daKmoR) [inline docs/blog/introducing-modern-web/index.md:4]: that should be longer as otherwise, it looks "boring" on the overview page

Comment (daKmoR) [reply docs/blog/introducing-modern-web/index.md]: reversed description it for now - if anyone has a better description feel free to suggest

File: docs/docs/test-runner/overview.md [hunk 2]
```diff
@@ -8,13 +8,14 @@ eleventyNavigation:
...
-📦&nbsp;&nbsp; Supports native es modules.<br>
-🔧&nbsp;&nbsp; Runs tests in parallel and isolation.<br>
+🔍&nbsp;&nbsp; Debug opens a real browser window with devtools.<br>
+🔧&nbsp;&nbsp; Exposes browser properties like viewport size and dark mode.<br>
+⏱&nbsp;&nbsp;Runs tests in parallel and isolation.<br>
 👀&nbsp;&nbsp; Interactive watch mode.<br>
-🏃&nbsp;&nbsp; Reruns only changed tests.<br>
-🚀&nbsp;&nbsp; Powered by [esbuild](../../dev-server/plugins/esbuild.md) and [rollup plugins](../../dev-server/plugins/rollup.md)
+🏃&nbsp;&nbsp; Fast development by rerunning only changed tests.<br>
+🚀&nbsp;&nbsp; Powered by [esbuild](https://modern-web.dev/docs/dev-server/esbuild.md) and [rollup plugins](https://modern-web.dev/docs/dev-server/rollup.md)
```
Comment (daKmoR) [inline docs/docs/test-runner/overview.md:18]: why use full urls (incl. `https://modern-web.dev`)?

Comment (LarsDenBakker) [reply docs/docs/test-runner/overview.md:18]: this was a copy paste from the blog post which had it like that

File: docs/blog/introducing-modern-web/index.md [hunk 3]
```diff
@@ -251,8 +273,52 @@ See more instructions in the [typescript-jsx-esbuild](https://modern-web.dev/lea
...
+};
+```
+
+This is only one example and plugins can do way more and if you want to go even more low level you can also write your own koa middleware.
 
 See more instructions in the [writing-plugin](https://modern-web.dev/learn/test-runner/writing-plugin/) learn section.
 
+## Thanks for reading
+
+We are incredible proud of our first Modern Web Tool 🤗 &nbsp; and we do hope you find it useful as well.
```
Comment (LarsDenBakker) [inline docs/blog/introducing-modern-web/index.md:317]:
```suggestion
We are incredibly proud of our first Modern Web Tool 🤗 &nbsp; and we do hope you find it useful as well.
```

File: docs/blog/introducing-modern-web/index.md [hunk 4]
```diff
@@ -251,8 +273,52 @@ See more instructions in the [typescript-jsx-esbuild](https://modern-web.dev/lea
...
+If you find an issues or you are stuck [please let us know](https://github.com/modernweb-dev/web/issues/new).
```
Comment (LarsDenBakker) [inline docs/blog/introducing-modern-web/index.md:318]:
```suggestion
If you find an issues or if you are stuck [please let us know](https://github.com/modernweb-dev/web/issues/new).
```

Comment (daKmoR) [reply docs/blog/introducing-modern-web/index.md]: fixed

File: docs/blog/introducing-modern-web/index.md (hunk 3 above)
Comment (daKmoR) [reply docs/blog/introducing-modern-web/index.md]: fixed

File: docs/docs/test-runner/overview.md (hunk 2 above)
Comment (daKmoR) [reply docs/docs/test-runner/overview.md]: fixed everywhere

**Pull request merged**